gdx/src/com/badlogic/gdx/graphics/g2d/SpriteBatch.java

```diff
@@ -36,6 +36,7 @@
  * @author mzechner
  * @author Nathan Sweet */
 public class SpriteBatch implements Batch {
+	public static final int MAX_SPRITE_COUNT = 16383;

```

Comment:
I like that this is in a constant field. Maybe it could have some snippet of Javadocs?
```java
/** The maximum number of sprites that can be rendered in a single batch. */
```

Reply:
I've added lengthy JavaDoc

Reply:
Perfect, it explains why the number is what it is.